sdk/storage/storage-file-share/samples/javascript/run.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+const { main: advancedMain } = require("./advanced");
+const { main: anonymousCredMain } = require("./anonymousCred");
+const { main: basicMain } = require("./basic");
+const { main: customPipelineMain } = require("./customPipeline");
+const { main: iteratorsFilesAndDirectoriesMain } = require("./iterators-files-and-directories");
+// const { main: iteratorsHandlesMain } = require("./iterators-handles");
+const { main: iteratorsSharesMain } = require("./iterators-shares");
+// const { main: proxyAuthMain } = require("./proxyAuth");
+const { main: sharedKeyCredMain } = require("./sharedKeyCred");
+const { main: withConnStringMain } = require("./withConnString");
+
+const { config: dotenv } = require("dotenv");
+dotenv();
+
+const samples = [
+  ["advanced", advancedMain],
+  ["anonymousCred", anonymousCredMain],
+  ["basic", basicMain],
+  ["customPipeline", customPipelineMain],
+  ["iterators-files-and-directories", iteratorsFilesAndDirectoriesMain],
+  // ["iterators-handles", iteratorsHandlesMain], // Must be populated with a pre-existing share/directory name to run
```

Comment:
Yes but it would still require that it already exists, so we would have to then have some external way to mask it out for CI. I think a better solution is to modify the sample so that it doesn't rely on any pre-existing resources.

Reply:
I'm not understanding this bit:

>  I think a better solution is to modify the sample so that it doesn't rely on any pre-existing resources.

This is _true_ but by the time the sample runs how/when the resource was created doesn't really matter, only that the environment variables point to the right thing. So it still seems like having those values specified in the sample.env makes sense to me.

SDK developers will create the identity once, the storage account once and just use that over and over (via .env)
CI will create the identity multiple times, the storage account multiple times and just use that over and over (by injecting environment variables directly).

Or am I missing how you want to do this?

Reply:
All of the rest of the examples in the storage packages create and tear down their resources dynamically, and the only thing they need is an account. I think we should just have this sample do the same thing. What I'm going to do for now is just make the samples exit if they aren't configured, and so this one will run fine in the runner if the environment variables aren't set up. It'll just log a message and skip itself. I'll add the variables to sample.env for customers' sake, but our CI shouldn't have to provision a share and a directory with some open handles to run this sample since that's a lot for the CI harness to have to set up and I'm not even sure if it could do that.